quick_start/my_text_classifier_bert.jsonnet

```diff
@@ -0,0 +1,48 @@
+local bert_model = "bert-base-uncased";
+
+{
+    "dataset_reader" : {
+        "type": "classification-tsv",
+        "tokenizer": {
+            "word_splitter": "bert-basic"
+        },
+        "token_indexers": {
+            "bert": {
+                "type": "bert-pretrained",
+                "pretrained_model": bert_model,
+            }
+        }
+    },
+    "train_data_path": "data/movie_review/train.tsv",
+    "validation_data_path": "data/movie_review/dev.tsv",
+    "model": {
+        "type": "simple_classifier",
+        "embedder": {
+            "allow_unmatched_keys": true,
```

Comment:
I finished https://github.com/allenai/allennlp/pull/3597 yesterday; once that's merged, this can go away.

Reply:
OK, I'm putting this PR on hold until that gets merged

Reply:
I'd just remove these two keys now.  That PR is going to be merged in the next hour.